server/app/views/components/ProgramCardFactory.java

```diff
@@ -134,23 +134,7 @@ private DivTag renderProgramRow(
             StyleUtils.hover(Styles.BG_GRAY_100),
             StyleUtils.joinStyles(extraStyles))
         .with(
-            p().withClasses(
-                    badgeBGColor,
-                    badgeFillColor,
-                    Styles.ML_2,
-                    StyleUtils.responsiveXLarge(Styles.ML_8),
-                    Styles.FONT_MEDIUM,
-                    Styles.ROUNDED_FULL,
-                    Styles.FLEX,
-                    Styles.FLEX_ROW,
-                    Styles.GAP_X_2,
-                    Styles.PLACE_ITEMS_CENTER,
-                    Styles.JUSTIFY_CENTER)
-                .withStyle("min-width:90px")
```

Comment:
Did the "min-width" get lost when pulling out into a common class? I think this was to prevent small screen sizes from collapsing the badge into a cramped state.

Reply:
I replaced it with "width: 100px". Having fixed width is better as otherwise they have width few pixels off and you can see elements misaligned.